prqlc/Taskfile.yaml

```diff
@@ -0,0 +1,68 @@
+version: "3"
+
+includes:
+  bindings-python:
+    taskfile: ./bindings/python
+    dir: ./bindings/python
+
+vars:
+  packages_core: -p prqlc-ast -p prqlc-parser -p prql-compiler -p prqlc
+  packages_addon: -p prql-compiler-macros -p compile-files
+  packages_bindings:
+    -p prql -p prql-java -p prql-js -p prqlc-clib -p prql-python
+
+tasks:
+  fmt:
+    desc: Format prqlc source files
+    cmds:
+      - cmd: |
+          # remove trailing whitespace
+          rg '\s+$' --files-with-matches --glob '!*.rs' . \
+          | xargs -I _ sh -c "echo Removing trailing whitespace from _ && sd '[\t ]+$' '' _"
+
+      - cmd: |
+          # rustfmt
+          cargo fmt {{.packages_core}} {{.packages_addon}} {{.packages_bindings}}
+
+      - cmd: |
+          # no dbg
+          rg 'dbg!' --glob '*.rs' . --no-heading
```

Comment:
100% fine to have these. We do have some of the fast ones like the greps & trailing whitespace in `pre-commit`, just as an FYI (having both is reasonable)